provider/resources.go

```diff
@@ -3956,6 +3959,15 @@ func Provider() tfbridge.ProviderInfo {
 				"pulumi": ">=2.15.0,<3.0.0",
 			},
 		},
+		Golang: &tfbridge.GolangInfo{
+			ImportBasePath: filepath.Join(
```

Comment:
Curious why you're mixing `filepath.Join` with the Sprintf string construction? Since this is a Go import path, I think you can just construct the whole thing with `/`

Reply:
> Curious why you're mixing `filepath.Join` with the Sprintf string construction? Since this is a Go import path, I think you can just construct the whole thing with `/`

Reasoning was to accommodate for the case where the major version was < 1 - in which case the versionString would be empty. `filepath.Join` takes care of all of those and produces a clean path.